geniza/entities/views.py

```diff
@@ -176,27 +179,105 @@ class PersonListView(ListView):
     # Translators: description of people list/browse page
     page_description = _("Browse people present in Geniza documents.")
     paginate_by = 50
+    form_class = PersonListForm
+    applied_filter_count = 0
+
+    # fields to facet
+    facet_fields = ["gender", "role__name", "persondocumentrelation__type__name"]
```

Comment:
Just to make sure I'm understanding, you're doing db-only search and faceting? If so, seems like it's worth a brief comment to make that explicit.

Did you decide that would be easier than implementing with solr? Or is it because you're only filtering and not doing keyword search?

Reply:
Exactly the latter—only filtering, so a system with analysis and indexing didn't seem necessary. I didn't actually consider using solr! Maybe it could have saved time.

As for the other piece of the model choice field, I worked a bit on that and I seem to essentially end up at the same number of queries. I might be missing something, but it didn't feel worth it to spend any more time on. But I'll definitely revisit if it has performance problems, so thank you for the idea and the link to ppa-django!